### PR TITLE
Add php 7.4 to travis build matrix

### PR DESCRIPTION
**Pull request by Xerkus**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### Description
Add php 7.4 to travis builds and move CS checks to php 7.3 from security only 7.2

### CODE REVIEW

**Comment by michalbundyra** [comment]
@Xerkus to solve the problem with PHP 7.4 lowest please add:
```json
    "conflict": {
        "phpspec/prophecy": "<1.10.2"
    },
```

into `composer.json`.

**Comment by Xerkus** [comment]
Can't add it as `conflict` is not dev only and it will affect users of the package.

File: composer.json [hunk 1]
```diff
@@ -40,9 +40,9 @@
         "laminas/laminas-modulemanager": "^2.8",
         "laminas/laminas-serializer": "^2.9",
         "laminas/laminas-servicemanager": "^3.3.2",
-        "phpspec/prophecy": "^1.7.5",
+        "phpspec/prophecy": ">=1.10.2",
         "phpstan/phpstan": "^0.10.5",
-        "phpunit/phpunit": "^8.0"
+        "phpunit/phpunit": "^8.5.1"
```
Comment (michalbundyra) [inline composer.json]: why not 8.5.2 ? maybe also || ^9.0 ? ;-)

**Comment by Xerkus** [comment]
Weird coverage under phpunit 9

**Pull request merged**